chain/client/src/sync_jobs_actor.rs

```diff
@@ -12,6 +13,9 @@ use near_primitives::state_part::PartId;
 use near_primitives::state_sync::StatePartKey;
 use near_primitives::types::ShardId;
 use near_store::DBCol;
+use std::time::Duration;
+
+const RESHARDING_RETRY_TIME: Duration = Duration::from_secs(10);
```

Comment:
Would be cool to benchmark how long does it take to perform the snapshot compaction on RPC node and start resharding after it's finished. It would give us more predicable performance. Not for this PR for sure. 

Making that delay configurable may be needed for testing on smaller and quicker networks like in nayduck where you may want the whole epoch to be done in 10s or less.

Reply:
I feel like it should be simple enough to tweak this value/add it as a parameter later?

Reply:
2.5 hours to compact a snapshot on an RPC node: https://nearinc.grafana.net/goto/lzX5DiMSg?orgId=1